src/add-and-remove.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable linebreak-style */
+function addTask(activities, description) {
+  const newActivity = { description, index: activities.length, completed: false };
+  activities.push(newActivity);
+  return activities;
+}
+
+function deleteTask(activities, index) {
+  activities.splice(index, 1);
+  return activities;
+}
+
+function deleteAll(activities) {
+  const result = activities.filter((activity) => (activity.completed === false));
+  return result;
+}
```

Comment:
- [ ] One requirement has not been met here. Look at the quote below 👇🏼:

> Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

Take a look at the following screenshots in order:

| **Added some tasks** |
| --- |
| ![one](https://user-images.githubusercontent.com/45002654/130304283-ed0fa69a-6c71-4fec-8cbc-67314edffb9c.PNG) |

| **After deleting the `first task` and the `third task`** |
| --- |
| ![three](https://user-images.githubusercontent.com/45002654/130304355-599a04f6-9924-411b-b167-5f9c121ac896.PNG) |

Looking at the screenshots above, the final indexes should have been 0 and 1 respectively after deleting some tasks basing on where your numbering has started. **_DON'T look at the index references from the localStorage Array, those are the obvious default ones and are not under test in this task._** So, there needs to implement a function that will update the tasks indexes after a delete operation. To further break it down, deleting a task with an index value of 0 should make the next task's index be updated to 0 and the one after it 1, so on so forth. The order of 0,1,2,3... must remain true whatever happens to the data. Please work on this.